### DISCUSSION TITLE
Domain Registrar Changed by itself

### DISCUSSION THREAD

**Discussion by Taimoor-Tariq** [vercel/community#3451] [Help] [answered]
### Summary

I have a domain that I got through vercel, i recently wanted to change some nameservers but it says that the registrar is no longer vercel.

Domain name: taimoor.me

### Steps to Reproduce

N/A

**Comment by amyegan** [comment]
Hey @Taimoor-Tariq. Did you transfer the domain from another account? Or is it on the same account where you originally purchased it?

**Reply by Taimoor-Tariq** [reply]
its the same account

**Reply by Taimoor-Tariq** [reply]
this is my Vercel user id: "wdSPukoyiPzWxMo466QmKZBZ"

**Reply by smaeda-ks** [reply]
@Taimoor-Tariq While we investigate the root cause of this event, we've manually updated your name servers to the ones you were originally trying to update to (digital ocean). Could you confirm that?

**Reply by Taimoor-Tariq** [reply]
sure

**Reply by smaeda-ks** [reply, marked as answer]
@Taimoor-Tariq The UI status has also been corrected.
